### PR TITLE
default theme based colors for pane and etc ui elements.

### PR DESCRIPTION
**Pull request by MairwunNx**
default theme colors for dark theme: 

image

without my changes on dark theme, really not cool:

image

### CODE REVIEW

File: jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroDarkTheme.css [hunk 1]
```diff
@@ -128,6 +132,16 @@
...
 }
 
+/*******************************************************************************
+ *                                                                             *
+ * ToolBar                                                                     *
+ *                                                                             *
+ ******************************************************************************/
+
+.tool-bar {
+    toolbar_background_color: #1d1d1d;
```
Comment (dukke) [inline jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroDarkTheme.css]: We don't need this var. We can simply reuse the `background_color` var already defined.

File: jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroDarkTheme.css [hunk 2]
```diff
@@ -153,6 +167,7 @@
 
 .scroll-pane {
     background_color: #29292a;
+    scroll_view_background_color: #1d1d1d;
```
Comment (dukke) [inline jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroDarkTheme.css]: Again, we don't need this var. We can simply reuse the `background_color` var already defined.

File: jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroDarkTheme.css [hunk 3]
```diff
@@ -27,19 +27,23 @@
 
 @import "JMetroBase.css";
 
-.background{
+.background {
     -fx-background-color: #1d1d1d;
```
Comment (dukke) [inline jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroDarkTheme.css]: Make this be equal to the background_color var.
`-fx-background-color: background_color`

thanks.

File: jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroLightTheme.css [hunk 4]
```diff
@@ -39,6 +39,10 @@
     -fx-text-fill: black;
 }
 
+.root {
+    background_color: white;
```
Comment (dukke) [inline jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroLightTheme.css]: same thing as mentioned for the dark theme.

File: jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroLightTheme.css [hunk 5]
```diff
@@ -118,6 +122,16 @@
...
 }
 
+/*******************************************************************************
+ *                                                                             *
+ * ToolBar                                                                     *
+ *                                                                             *
+ ******************************************************************************/
+
+.tool-bar {
+    toolbar_background_color: white;
```
Comment (dukke) [inline jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroLightTheme.css]: same thing as mentioned for the dark theme.

File: jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroLightTheme.css [hunk 6]
```diff
@@ -143,6 +157,7 @@
 
 .scroll-pane {
     background_color: #e9e9e9;
+    scroll_view_background_color: white;
```
Comment (dukke) [inline jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroLightTheme.css]: same thing as mentioned for the dark theme.

**Comment by dukke** [comment]
Hey Pavel, nice work again. Good idea on setting background colors for other containers.

We're still missing setting background color for: 
- BorderPane 
- HBox
- VBox
- StackPane
- GridPane
- FlowPane
- TilePane
- AnchorPane

These classes don't have any styleclass applied by default, so we need to change the backgound color using the class selector, like this:
```
BorderPane {
  -fx-background-color: background_color
}
```

Finally a sample would be nice. Can be a sample showcasing all background colors on all containers. So a sample that has all containers in the same scene.

File: jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroDarkTheme.css (hunk 1 above)
Comment (MairwunNx) [reply jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroDarkTheme.css]: Not, background_color so dark! If i make background_color as color value for toolbar then it causes color unbalance :(

File: jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroDarkTheme.css (hunk 2 above)
Comment (MairwunNx) [reply jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroDarkTheme.css]: Not, background_color so dark! If i make background_color as color value for toolbar then it causes color unbalance :(

File: jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroDarkTheme.css (hunk 3 above)
Comment (MairwunNx) [reply jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroDarkTheme.css]: Answered upper 👍

File: jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroLightTheme.css (hunk 4 above)
Comment (MairwunNx) [reply jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroLightTheme.css]: Not fully understand for what  code you commented? For background_color variable?

File: jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroLightTheme.css (hunk 5 above)
Comment (MairwunNx) [reply jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroLightTheme.css]: Not fully understand for what code you commented? For background_color variable?

File: jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroLightTheme.css (hunk 6 above)
Comment (MairwunNx) [reply jmetro/src/main/resources/jfxtras/styles/jmetro8/JMetroLightTheme.css]: Not fully understand for what code you commented? For background_color variable?

**Comment by MairwunNx** [comment]
Sorry, i see what you want to set to some dark theme colors default as #111, but it not true, see please, with my changes it really dark theme, as chocolate with coffee 👍 

image

And see your **Not dark** - Black theme

image

**Comment by MairwunNx** [comment]
I think all conversations resolved, you can merge it PR 👍

**Comment by MairwunNx** [comment]
And! 

```
BorderPane
HBox
VBox
StackPane
GridPane
FlowPane
TilePane
AnchorPane
```

Not needed in styling, it nested just base Pane style, and it PR changing default colors for all controls, you also can check it self if you not trust me :DD

**Comment by dukke** [comment]
Missing sample test app for this feature.

**Comment by dukke** [comment]
This has been fixed but differently from suggested in this PR. ThemeTester app has been added and is a good place to check how this is working. 
Some suggestions that are here like changing toolbar and scrollpane have been taken into account.

**Pull request closed without merging**